service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/C2UploadedEventHandlerTest.java

```diff
@@ -72,7 +72,7 @@ void before() {
     @AfterEach
     void resetInvocations() {
         reset(notificationService);
-        reset(inboxLookupService);
+        reset(localAuthorityRecipients);
```

Comment:
are these resets needed? Only thought they were required when we had nested tests with verifications.